### ISSUE TITLE
Error when converting to savedsearches for Splunk backend : "AttributeError: 'NoneType' object has no attribute 'split'" in splunk.py

### ISSUE THREAD

**Issue by EhouarnBeriotOConnell** [SigmaHQ/sigma-cli#16]
Hello ! 
When trying to convert a sigma rule into savedseaches format i get an error from splunk backend (`backends/splunk/splunk.py`).
Conversion goes smoothly without the `-f savedsearches` flag or with `-f default`.
It looks like a bug, but maybe I am missusing the tool.
I am trying to dig deeper into the code to understand the error, but I am not too familiar with it yet.

|  module | version    
| :--------------- |:---------------:|
|pysigma|                             0.9.2
|pysigma-backend-splunk  | 1.0.2
|pysigma-pipeline-sysmon  |1.0.2
|pysigma-pipeline-windows |1.1.0
|sigma-cli|                             0.6.0

**Command**
```
sigma convert -t splunk -f savedsearches -p windows-audit sigma/rules/windows/process_creation/proc_creation_win_susp_whoami.yml
```

**Stack trace**
```
Traceback (most recent call last):
  File "/home/<user>/.local/bin/sigma", line 8, in <module>
    sys.exit(main())
  File "/home/<user>/.local/lib/python3.9/site-packages/sigma/cli/main.py", line 17, in main
    cli()
  File "/home/<user>/.local/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/<user>.local/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/<user>/.local/lib/python3.9/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/<user>/.local/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/<user>t/.local/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/<user>/.local/lib/python3.9/site-packages/sigma/cli/convert.py", line 147, in convert
    result = backend.convert(rule_collection, format)
  File "/home/<user>/.local/lib/python3.9/site-packages/sigma/conversion/base.py", line 97, in convert
    return self.finalize(queries, output_format or self.default_format)
  File "/home/<user>/.local/lib/python3.9/site-packages/sigma/conversion/base.py", line 356, in finalize
    return self.__getattribute__("finalize_output_" + output_format)(queries)
  File "/home/<user>.local/lib/python3.9/site-packages/sigma/backends/splunk/splunk.py", line 126, in finalize_output_savedsearches
    return f"\n[default]" + self._generate_settings(self.output_settings) + "\n" + "\n".join(queries)
  File "/home/<user>/.local/lib/python3.9/site-packages/sigma/backends/splunk/splunk.py", line 98, in _generate_settings
    output += f"\n{k} = " + " \\\n".join(v.split("\n"))  # cannot use \ in f-strings
AttributeError: 'NoneType' object has no attribute 'split'
```

(sorry for the multiple issues recently)

**Comment by thomaspatzke** [comment]
Back to CLI, because it's indeed a CLI issue 😂

**Comment by thomaspatzke** [comment]
The issue was caused by the CLI passing None as min/max-time if not provided, which overrides the default values from the backend.

Workaround: provide --min-time/--max-time parameters.

Fix will be a breaking change: moving to generic way to pass backend-specific parameters.

**Comment by EhouarnBeriotOConnell** [comment]
Problem should be fixed by [0.7.0](https://github.com/SigmaHQ/sigma-cli/releases/tag/v0.7.0), if I am not mistaking.
I will not be able to test the changes before some time, but I'll close for now and reopen if I encounter the problem again.
Thank you for providing an answer, a workaround, and then fixing it so quickly !
